dataeng/resources/remote-config.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+#
+# Fetch the latest studio and LMS configs for edx and edge.
+#
+# For each deployment:
+#
+# 1. Assume the cross-account role specifically made for this purpose.
+# 2. Fetch the encrypted configs.
+# 3. Login to Vault.
+# 4. Fetch the decryption keys from Vault.
+# 5. Decrypt the configs
+
+set -xe
+
+env | sort
+
+# Install dependencies early in this script.
+pip install asym-crypto-yaml
```

Comment:
I'm just curious about the other dependencies in this script, and where they come from.    Those include aws cli, vault, and jq.   Can we just assume these are on the Jenkins already?

Reply:
* awscli is installed on the machine
* jq: https://github.com/edx/configuration/pull/6056 (I manually installed it on the machine)
* vault... drat.  I guess I only tested the vault step on my local machine.

Reply:
https://github.com/edx/configuration/pull/6076